modules/gdscript/gdscript_editor.cpp

```diff
@@ -177,7 +177,7 @@ bool GDScriptLanguage::validate(const String &p_script, const String &p_path, Li
 			}
 		}
 		return false;
-	} else {
+	} else if (r_functions) {
```

Comment:
`r_functions` is optional and the method was missing check like for other lists.